src/pymental/meta.py

```diff
@@ -19,11 +19,13 @@ def __new__(cls, cls_name, bases, attrs):
                 remaining_attrs[attr] = value
 
         remaining_attrs['_fields'] = fields
-        return super().__new__(cls, cls_name, bases, remaining_attrs)
+        return super(MetaModel, cls).__new__(cls, cls_name, bases, remaining_attrs)
 
 
 # base for our custom objects (Job, Input, Profile, etc.) aka models.Model
-class Model(metaclass=MetaModel):
+class Model(object):
+    __metaclass__ = MetaModel
```

Comment:
Oh right, I did not consider p27 style might not be compatible with p3 metaclass style. Will change. Thanks.